lua/starfall/libs_sh/hologram.lua

```diff
@@ -293,14 +293,34 @@ else
 		end
 	end
 
+	local render_GetColorModulation, render_GetBlend = render.GetColorModulation, render.GetBlend
+	local render_SetColorModulation, render_SetBlend = render.SetColorModulation, render.SetBlend
+
 	--- Manually draws a hologram, requires a 3d render context
 	-- @client
-	function hologram_methods:draw()
+	-- @param bool? tint If true, renders the hologram with it's color and opacity. This is the default behavior.
+	function hologram_methods:draw(tint)
 		if not instance.data.render.isRendering then SF.Throw("Not in rendering hook.", 2) end
-
+		tint = tint == nil and true or tint
```

Comment:
The `and true` part isn't necessary since `tint == nil` will already resolve to `true`.
Also, I don't think it's necessary to reassign the `tint` variable since you're only gonna use it once in a very simple `if` statement.